<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EOGWindow</class>
 <widget class="QMainWindow" name="EOGWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EOGWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="topButtonsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="recordsLayoutVisibleButton">
        <property name="text">
         <string>Records list</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recordButton">
        <property name="text">
         <string>Record</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="recordsAndFiltersLayout">
      <item>
       <widget class="QWidget" name="recordsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="buttonsLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="changeFolderButton">
             <property name="text">
              <string>Change folder</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="recordsListWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="filtersLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="chooseFiltersButton">
          <property name="text">
           <string>Choose filters</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="chooseFiltersFrame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionClear_settings_and_exit"/>
    <addaction name="actionRestore_default_filter_configurations"/>
   </widget>
   <addaction name="menuOther"/>
  </widget>
  <action name="actionClear_settings_and_exit">
   <property name="text">
    <string>Clear settings and exit</string>
   </property>
  </action>
  <action name="actionReset_default_filter_configurations">
   <property name="text">
    <string>Reset default filter configurations</string>
   </property>
  </action>
  <action name="actionRestore_default_filter_configurations">
   <property name="text">
    <string>Restore default filter configurations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>recordsLayoutVisibleButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>recordsWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseFiltersButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>chooseFiltersFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>EOGWindow</receiver>
   <slot>requestStorageDirFromUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear_settings_and_exit</sender>
   <signal>triggered()</signal>
   <receiver>EOGWindow</receiver>
   <slot>clearSettingsAndExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestore_default_filter_configurations</sender>
   <signal>triggered()</signal>
   <receiver>EOGWindow</receiver>
   <slot>restoreDefaultFilterConfigurations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>requestStorageDirFromUser()</slot>
  <slot>clearSettingsAndExit()</slot>
  <slot>restoreDefaultFilterConfigurations()</slot>
  <slot>startRecording()</slot>
 </slots>
</ui>
